doc/proposals/scorecard-custom-tests.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Scorecard Custom Test Framwork
+authors:
+  - "@jemccorm"
+reviewers:
+  - TBD
+  - "@joelandford"
+  - "@zeus"
+approvers:
+  - TBD
+  - "@shurley"
+  - "@dmesser"
+creation-date: 2020-03-04
+last-updated: 2020-03-04
+status: implementable
+---
+
+# Scorecard - Custom Tests
+
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created in [operator-sdk/doc][operator-sdk-doc]
+
+## Summary
+
+This enhancement proposal outlines the SDK scorecard’s design for running of custom tests, specifically tests that are outside of what Red Hat includes by default within the scorecard’s tests.   The proposed design would embed the KUDO test framework into the SDK scorecard and leverage it for an actual implementation, wrapping Kuttl where necessary to provide a good SDK experience to the end-user.
+
+## Motivation
+
+The scorecard needs a robust means of enabling end-users of the SDK to write and execute their own operator tests.   Ideally, the custom test framework would decouple test logic from within the SDK itself, making the SDK and scorecard logic in particular more flexible and maintainable in the long run.
+
+## Goals
+
+ * Document a design for scorecard to execute end-user or custom tests.
+
+ * Document stories/steps required to deliver scorecard custom test functionality.
+
+### Non-Goals
+
+ * This is a design proposal so the sample source code referenced here is not production quality and doesn’t reflect the actual proposed implementation.  Sample tests were constructed for this proposal only to prove out certain aspects of the design.
+
+## Proposal
+
+This design proposal is to integrate the Kuttl test framework into the Operator SDK scorecard specifically for enabling the creation of custom tests that the scorecard could execute and report on.
```

Comment:
See: https://github.com/kudobuilder/kuttl
So, some cons in POV
- It has just 3 contributors / 0 forks /  5 starts
- I could not find any doc on/over the project. 

IHMO it would not be mature and adopted enough for we are able to ensure that it will be kept and maintained. In this way, it also would probably mean that we should need to maintain this project as well without have the knowledge and control over it.